serde/src/test/org/apache/hadoop/hive/serde2/avro/TestTypeInfoToSchema.java

```diff
@@ -261,7 +261,7 @@ public void createAvroDateSchema() {
   public void createAvroTimestampSchema() {
     final String specificSchema = "{" +
       "\"type\":\"long\"," +
-      "\"logicalType\":\"timestamp-millis\"}";
+      "\"logicalType\":\"timestamp-micros\"}";
```

Comment:
Its better to extend test classes one for each of these datatypes, rather than replace this by TIMESTAMP_TYPE_NAME_MILLIS.

Reply:
We needed a change here as our avro schemas in hive by default are now serialized to have microsecond precision.

Reply:
Here we are testing the schema conversion and hence, according to the current architecture, the logical type of schema is the maximum precision of timestamp that can be provided i.e, right now it is micros